source/extensions/filters/http/aws_lambda/aws_lambda_filter.cc

```diff
@@ -265,6 +269,10 @@ bool AWSLambdaFilter::isTransformationNeeded() {
   return functionOnRoute() != nullptr && (functionOnRoute()->unwrapAsAlb() || functionOnRoute()->hasTransformerConfig());
```

Comment:
nit: Could we update this method to be more explicit: isResponseTransformationNeeded? Now that we have a seprate request and response transformation, I fear it will be easy to misuse this function without understanding its designed for response transformations

Reply:
Yeah, definitely makes sense!

Reply:
Renamed the `isTransformationNeeded` to `isResponseTransformationNeeded` here: cb39433a0958a4bec1cae76b686fe1fbbe5ec390